README.md

```diff
@@ -1,26 +1,30 @@
 # go-libp2p-loggables
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
-[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/project-libp2p-blue.svg?style=flat-square)](http://github.com/libp2p/libp2p)
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
 [![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
 > logging helpers for go-libp2p
 
-## Documenation
+## Install
 
-See https://godoc.org/github.com/ipfs/go-libp2p-loggables.
+```
+go get github.com/libp2p/go-libp2p-loggables
+```
```

Comment:
Should this be compiled from source? For instance:

```
git clone git://github.com/libp2p/go-libp2p-loggables.git
cd go-libp2p-loggables && go build
```

I looked around at a few libraries on [this list](http://go-lang.cat-v.org/pure-go-libs), and they all seem to think that the installation should be `go get`. What am I missing?